api/v1alpha1/instancegroup_types.go

```diff
@@ -87,6 +88,8 @@ const (
 
 	DockerRuntime     ContainerRuntime = "dockerd"
 	ContainerDRuntime ContainerRuntime = "containerd"
+
+	UpgradeLockedAnnotationKey = "instancemgr.keikoproj.io/lock-upgrades"
```

Comment:
Can we move this to `eks.go` with other annotations?
https://github.com/keikoproj/instance-manager/blob/master/controllers/provisioners/eks/eks.go#L36-L43

Reply:
How important is this move? a direct approach results in a circular dependency between v1alpha1 and eks - could try and rework it, but it seems non-trivial.

Reply:
Yeah, we can leave there in that case, it was just a nit